omod/src/main/java/org/openmrs/module/appointments/web/controller/AdhocTeleconsultationController.java

```diff
@@ -0,0 +1,36 @@
+package org.openmrs.module.appointments.web.controller;
+
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.appointments.service.impl.TeleconsultationAppointmentService;
+import org.openmrs.module.appointments.model.AdhocTeleconsultationResponse;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/adhocTeleconsultation")
```

Comment:
do we need another controller for this? Why not just use existing appointmentscontroller?